spec/requests/security_activity_spec.rb

```diff
@@ -19,13 +19,16 @@
     end
 
     expect_event SecurityActivity::ACCOUNT_LOCKED
+    expect_event_on_security_page SecurityActivity::ACCOUNT_LOCKED
   end
 
   it "records MANUAL_ACCOUNT_UNLOCK events" do
     unlock_token = user.lock_access!
     get user_unlock_path(unlock_token: unlock_token)
+    user.reload
```

Comment:
What's this needed for?

Reply:
We need to reload the user from the database here since the cached `user` doesn't get updated when the GET request is made to unlock the account.